example-facets/films/filters.py

```diff
@@ -13,3 +19,21 @@ def configure_facets(self):
         self.filters["release_year"].facet = Facet()
         self.filters["rating"].facet = Facet()
         self.filters["listed_in"].facet = Facet()
+
+    def filter_search(self, queryset, name, value):
+        vector = search.SearchVector("description", weight="B")
+        query = search.SearchQuery(value)
+        return (
+            queryset.annotate(
+                rank=search.SearchRank(vector, query),
+                headline=search.SearchHeadline(
+                    "description",
+                    query,
+                    min_words=30,
+                    start_sel="<mark>",
+                    stop_sel="</mark>",
+                ),
```

Comment:
This portion is useful if you want to extract and show what block of text matched the search term. Do we want to show it in the table? If not, we can probably remove it.